net-p2p/qbittorrent/qbittorrent-5.0.0.ebuild

```diff
@@ -13,45 +13,32 @@ if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 else
 	SRC_URI="
-		https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz
-		verify-sig? ( https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz.asc )
+		https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz
+		verify-sig? ( https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz.asc )
 	"
-	KEYWORDS="amd64 ~arm ~arm64 ~ppc64 ~riscv x86"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
 
 	BDEPEND="verify-sig? ( sec-keys/openpgp-keys-qbittorrent )"
 	VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/qBittorrent.asc
 fi
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
```

Comment:
Thanks for spotting this!

The gpl.html file in the sources says that qbittorrent's source code is licensed GPL-2+ but also says that the image assets are GPL-3+, so now that we are fixing this, maybe we should list both.